Rename YAML file

alexpapworth opened the pull request:
We added customer-api-docs recently, so got our Rails app to output two different YAML files, depending on which Docs repository it's going to.

Also added in tags to group the requests into different categories.

alexpapworth committed:
Rename openapi.yaml to resellers_api.yaml

Renaming this so we can copy files between repositories without needing to modify anything.

alexpapworth committed:
Add latest reseler_api.yaml

Regenerating now we've added tags to separate the requests into different sections. Either "Config" or "API" for now, for if it's a real request or just key configuration.

alexpapworth commented on resellers_api.yaml:
```diff
@@ -1,83 +1,280 @@
+                - name: Alexander Fleming House
+                  timezone: London
+                  regulator_reference: CQC45678
+                  postcode:
```
Not sure why these postcodes are coming up as blank now. Do the tests not work anymore?

SimonMerrikin replied:
`/requests/resellers_spec.rb` doesn't have seeded postcodes. `/controllers/resellers_controller_spec.rb` does, but I see now that that file isn't the one read for building the yaml?

So we need to make sure seeded data we want to see is in `spec/requests/resellers_spec.rb`

alexpapworth replied:
Ah I see. I'll update the request specs :+1:

alexpapworth committed:
Add bin/run script

Just a handy helper to run the Swagger site locally. Easier to remember `bin/run` than it is to check the README for the `ruby -run -e httpd . -p 5000` line.

SimonMerrikin commented:
I assume we should merge this. Working off this branch for PR https://github.com/digital-reception/reseller-api-docs/pull/10

alexpapworth commented:
Ah yep, good point. Merging now.
